tests/network/localnet/migration_stuntime/libstuntime.py

```
@@ -0,0 +1,148 @@
+    Args:
+        ping_log: Tail output from ping including summary lines.
+
+    Returns:
+        float: Stuntime in seconds.
+
+    Raises:
+        InsufficientStuntimeDataError: When summary line with transmitted/received is missing.
+    """
+    summary_match = re.search(r"(\d+)\s+packets transmitted,\s+(\d+)\s+received", ping_log)
+    if not summary_match:
+        raise InsufficientStuntimeDataError("Insufficient data to compute stuntime (missing ping summary line)")
```

Ruff (0.15.10)

[warning] Avoid specifying long messages outside the exception class

(TRY003)